src/backend/preprocessor/Errors.js

```diff
@@ -22,6 +22,12 @@ define(function () {
 	this.message = name + ': NAME UNKNOWN';
     }
 
+    function Overflow (type, node) {
+	this.type = type;
+	this.location = node.loc;
+	this.message = type + ' OVERFLOW';
```

Comment:
Technically, an overflow can only occur after an arith operation. This should be called sth like "integer constant not in valid range"